pandas/_libs/tslibs/src/datetime/np_datetime_strings.c

```diff
@@ -67,49 +67,59 @@ This file implements string parsing and creation for NumPy datetime.
  * Returns 0 on success, -1 on failure.
  */
 
+enum DatetimePartParseResult {
```

Comment:
If you use `typedef` here you don't need to repeat `enum` every time you refer to this type

Reply:
nice, thanks!